src/qengine/state/operators.cpp

```diff
@@ -960,8 +957,8 @@ bitCapInt QEngineCPU::IndexedADC(bitLenInt indexStart, bitLenInt indexLength, bi
     }
 
     // We calloc a new stateVector for output.
-    Complex16* nStateVec = AllocStateVec(maxQPower);
-    std::fill(nStateVec, nStateVec + maxQPower, Complex16(0.0, 0.0));
+    complex* nStateVec = AllocStateVec(maxQPower);
+    std::fill(nStateVec, nStateVec + maxQPower, complex(0.0, 0.0));
```

Comment:
I really wonder whether or not all of these std::fill’s are actually necessary.  Aren’t the nStateVec’s largely populated during the par_for comprehensively?  If so, these can be removed - that would be a change for another PR however.

Comment was applied at random, not specific to this particularly instance.